### PR TITLE
IL/WI/SD Changes from Highway63

### PR DESCRIPTION
**Pull request by Markkos1992**
Illinois IL 108: Split route at Kampsville Ferry. Long segment now ends at point KamFry. Short segment renamed IL108Kam. All users with this segment must re-enter routes.  (https://forum.travelmapping.net/index.php?topic=4160.0)
Illinois Il 108 (Kampsville): Created route.
Illinois IL 100: Recentered IL108.

Wisconsin WI 113: Added CRDM, PleSt, CRK. Recentered WI19_S at roundabout. Renamed CRP/V_Dan->CRP/V. Split CRJ_Col into CRJCol_W and CRJ_ColE. Split route at Merrimac Ferry. South segment now ends at point MerFry. North segment renamed WI113Bar. All users with this segment must re-enter routes.  (https://forum.travelmapping.net/index.php?topic=5155.0)
Wisconsin WI 113 (Baraboo): Created route.
Wisconsin WI 19: Recentered WI113_E at roundabout.
Wisconsin WI 188: Recentered WI113.
Wisconsin WI 78: Recentered WI113_S.
Wisconsin US 12 Baraboo: Recentered WI113_S. Deleted unused alt labels.

South Dakota SD 258: Recentered I-90_W, which has been renamed I-90.  (https://forum.travelmapping.net/index.php?topic=5156.0)
South Dakota I-90: Recentered exit 308 (SD 258).

Also removed WI123 alternate name and other unused alternate labels.

### CODE REVIEW

**Comment by Markkos1992** [comment]
Datacheck successful after continually updating the usawi and usawi_con files.

File: hwy_data/WI/usausb/wi.us012busbar.wpt [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 WI33/136 +US12_W http://www.openstreetmap.org/?lat=43.474660&lon=-89.768772
 WI33_E http://www.openstreetmap.org/?lat=43.474738&lon=-89.744225
 5thAve +WI113 http://www.openstreetmap.org/?lat=43.471826&lon=-89.744182
-WI113_S http://www.openstreetmap.org/?lat=43.467240&lon=-89.744396
-CRDL +WI123_S http://www.openstreetmap.org/?lat=43.463043&lon=-89.746070
-WalSt http://www.openstreetmap.org/?lat=43.460458&lon=-89.751348
-WI136 +US12_E http://www.openstreetmap.org/?lat=43.459959&lon=-89.768858
+WI113_S http://www.openstreetmap.org/?lat=43.467299&lon=-89.744500
+CRDL http://www.openstreetmap.org/?lat=43.463043&lon=-89.746070
+WalSt http://www.openstreetmap.org/?lat=43.460458&lon=-89.751348a
```
Comment (rickmastfan67) [inline hwy_data/WI/usausb/wi.us012busbar.wpt:11]: Looks like datacheck needs a new check.  This should have been flagged in it.

Comment (michihdeu) [reply hwy_data/WI/usausb/wi.us012busbar.wpt:11]: Really? Why should anything be flagged when there is no issue?

Comment (rickmastfan67) [reply hwy_data/WI/usausb/wi.us012busbar.wpt:11]: Because the line is broken, and the point doesn't show up in the file on the site?

**Pull request merged**